src/legacy/core_plugins/expressions/public/index.scss

```diff
@@ -0,0 +1,13 @@
+// Import the EUI global scope so we can use EUI constants
+@import 'src/legacy/ui/public/styles/_styling_constants';
```

Comment:
This is how we've been handling Styles throughout and yes the constants aren't currently being used, but if we remove this line and then someone tries to add `padding: $euiSize` their styles won't compile anymore and most likely they won't know why.

Reply:
@cchaos I see, thanks. Our use of CSS in Kibana really needs improvement. For example, EUI SASS constants could be automatically added to all plugins.

Reply:
ok, but this would prevent us from moving the service to the new platform, which was planned for 7.6 and other teams depend on this for their migration. can we put this styles somewhere else for now ?

Reply:
@streamich We agree but this is what the platform team has provided us when we asked to have SASS support so we can remove LESS (which even that still exists ☹️ ), and on the EUI side we're starting to make efforts to modularize so it's also not a massive library to compile. 

I'll leave this now for the author of the plugin to decide how to handle this.

Reply:
OMG 😱 didn't know we have LESS 😱